packages/babel-preset-expo/src/index.ts

```diff
@@ -98,6 +98,21 @@ function babelPresetExpo(api: ConfigAPI, options: BabelPresetExpoOptions = {}):
     extraPlugins.push(require('@babel/plugin-transform-parameters'));
   }
 
+  if (!isDev && hasModule('metro-transform-plugins')) {
```

Comment:
Why should this not run in development, too?

Reply:
We don't do any minification in development to keep things fast, might change in the future but right now it doesn't provide much benefit.